src/main.rs

```diff
@@ -100,6 +102,12 @@ pub async fn main() {
                 node_public_key.clone().into(),
             )));
 
+            let secret_key = ckb_config.read_secret_key().unwrap();
+            let secp = Secp256k1::new();
```

Comment:
Using `secp256k1_instance` in `src/fiber/types.rs` is better, because that function only initialize secp251k1 instance once and keep it in memory (there are many places we need to use the secp256k1 instance).